player.c

```diff
@@ -1967,6 +1967,19 @@ void Link_HandleSwordCooldown() {  // 879ac2
 
 }
 
+static int GetCurrentItemButtonIndex() {
+  if (enhanced_features0 & kFeatures0_SwitchLR) {
+    return (joypad1L_last & kJoypadL_X) ? 1 :
+      (joypad1L_last & kJoypadL_L)  ? 2 :

```

Comment:
double indentation on newlines (or align the (-characters)

Reply:
Don't duplicate those functions. Put them in hud.c and remove the word 'static'.